app/assets/javascripts/spree/frontend/main.js

```diff
@@ -2,7 +2,7 @@
 function Spree () {}
 
 Spree.ready = function (callback) {
-  return jQuery(document).on('page:load turbo:load', function () {
+  return jQuery(document).on('DOMContentLoaded page:load turbo:load', function () {
```

Comment:
Not sure this change is needed anymore after https://github.com/spree/spree_legacy_frontend/pull/21